features/command_status.feature

```diff
@@ -78,6 +78,17 @@ Feature: status command
       And the output should match /untracked: *\[4\] *untracked_file/
 
 
+  Scenario: Status properly reports all file changes w/ symlink
+    Given I am in a complex working tree status matching scm_breeze tests
+    When I cd to ".."
+      And I run `sh -c 'ln -s mygitrepo symlink && cd symlink && scmpuff status'`
```

Comment:
Using `sh -c` rather than `When I cd to symlink` because https://github.com/cucumber/aruba/blob/0cbd0e826994a67259b242c7befdb956552246e8/lib/aruba/api/core.rb#L47-L107 resolves symbolic links, thus defeating the purpose of the test.